samples/05_content_publishers/pdf_table_to_pdf_map.ipynb

```diff
@@ -34,7 +34,7 @@
     "\n",
```

Comment:
<p>Can we switch back to using a profile?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
This now uses _gis = GIS('home')_